VoltAir/Engine/android/deploy/src/org/literacyapp/voltair/VoltAirActivity.java

```diff
@@ -424,7 +428,20 @@ public void studentUpdateReceiver(String availableLetters) {
         if (availableLetters != null) {
             Log.i(LOG_TAG, availableLetters);
         }
+
+        SharedPreferences settings = getSharedPreferences(VOLTAIR_PREFS, Context.MODE_PRIVATE);
+        settings.edit().putString(PREF_STUDENT_LETTERS, availableLetters).commit();
+
         onStudentUpdateReceiver(availableLetters);
     }
 
+    public String getValidLetters() {
+        String letters = "AB";
```

Comment:
@diegowald Change so that the following list of letters are used for English and Swahili:

- English: ["e", "t", "a"]
- Swahili: ["a", "i", "u"]